src/style.css

```diff
@@ -0,0 +1,123 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.form {
+  display: flex;
+  margin: 20px auto 0;
+  justify-content: center;
+
+}
+
+.form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.form_small {
+  width: 100%;
+  height: 42px;
+}
+
+.form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.form__label_big::after {
+  content: "";
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  position: absolute;
+  display: block;
+}
+
+.form__label_small::after {
+  content: "";
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  position: absolute;
+  display: block;
+}
+
+.form__input {
+  width: 100%;
+  height: 100%;
+  outline: none;
+  font-family: inherit;
+  padding-right: 0;
+  flex-shrink: 0;
+  color: #3d4e61;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.form__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 600;
+}
+
+.form__input::placeholder {
+  color: inherit;
+  font-weight: 300;
+}
+
+.form__input_big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.form__input_small::placeholder {
+  font-size: 14px;
+  line-height: 22px;
+}
+
+.form__input_big {
```

Comment:
you could write it w/o `_big` or `_small` selectors. For example, write all styles for small input (and common styles) inside `.form__input` selector. Then in `.form__input_big` **overwrite** styles you needed (it's described in the checklist).

Anyway, very good job - approved 👍